.github/workflows/ci-test.yaml

```diff
@@ -8,24 +8,24 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: 1.15
-      - name: make local-test
+      - name: Run unit tests
         env:
           VERBOSE: true
         run: |
-          make local-test
+          source ./scripts/build.sh; unit_local
   stress-test-linux:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: make stress-test
+      - name: Run stress tests
         env:
           VERBOSE: true
         run: |
-          make stress
+          source ./scripts/build.sh; stress
   integration-test-on-kind:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: run tests
         run: |
-          SONOBUOY_CLI=../../build/linux/amd64/sonobuoy ./scripts/run_integration_tests.sh
+          source ./scripts/build.sh; prepare_and_run_integration
```

Comment:
Using source for now; will probably add a handler for arguments so that we can call it directly. I ran into a silly problem where I was testing this all locally and my terminal kept closing. But it was because of the source & set -e, then on failure my own terminal would exit :) Took me longer than necessary to figure that one out.

I'll go ahead and start working on the handler code, it can be in this PR or the next depending on when you review/when I'm done.